.github/workflows/ci.yml

```diff
@@ -316,6 +316,7 @@ jobs:
       - name: Value Entity Archetype
         run: |-
           export SDK_VERSION=$(cat ~/kalix-sdk-version.txt)
```

Comment:
Is this version file still there? Alternatively, this should reference `steps.build_sdk.outputs.sdk_version`, right?